nix-tools-default.nix

```diff
@@ -17,10 +19,17 @@ let  nix-tools = import nix-tools-pkgs-path {
   inherit pkgs;
   # the iohk-module contains cross compilation specific patches
   inherit (commonLib.nix-tools) iohk-module iohk-extras;
-};
+  };
+  # Allow hsPkgs to be returned separately (useful when using
+  # callCabalPlanToNix and callStackToNix on Hydra)
+  hsPkgs = nix-tools.hsPkgs or nix-tools;
 in {
     _lib = commonLib;
 
+    # Unlike `nix-tools._raw`, accessing `nix-tools-raw` does not force
+    # the components to be enumerated.
+    nix-tools-raw = nix-tools;
```

Comment:
shouldn't this be `nix-tools-raw = hsPkgs`?

Reply:
We need to be able to return something other than `hsPkgs` from `nix-tools-pkgs` and get access to it through `nix-tools-raw`.  In particular the nix file generated by callCabalPlanToNix.  In the case that `nix-tools-pkgs` returns both `plan-nix` and `hsPkgs` we need to be able to access both via `nix-tools-raw.plan-nix` and `nix-tools-raw.hsPkgs`.  That way we can get the `plan-nix` without ever causing it to be imported (by looking at `hsPkgs`).